azure-pipelines.yml

```diff
@@ -6,6 +6,9 @@
 trigger:
 - master
 
+pr:
+- master
+
```

Comment:
[I think this might actually cause it to trigger on PRs to master](https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml#pr-triggers)